.github/workflows/ci-ai-robots-txt.yml

```diff
@@ -16,7 +16,7 @@ jobs:
       - uses: actions/checkout@v4
       - uses: actions/setup-node@v4
         with:
-          node-version: 20.x
+          node-version: 22.x
```

Comment:
`--experimental-strip-types` is available in Node 22, which is useful for running the `getrobotstxt` script 👀 

We should probably look at bumping the node version for all packages in the monorepo at some point 🤔